pkg/updater/resultstore/resultstore_test.go

```diff
@@ -1132,6 +1133,176 @@ func TestProcessGroup(t *testing.T) {
 	}
 }
 
+func TestGetTestResults(t *testing.T) {
+	cases := []struct {
+		name      string
+		testsuite *resultstore.TestSuite
+		want      []*resultstore.Test
+	}{
+		{
+			name: "empty test suite",
+			testsuite: &resultstore.TestSuite{
+				SuiteName: "Empty Test",
+				Tests:     []*resultstore.Test{},
```

Comment:
if you remove the check on line 463 in `resultstore.go` as I suggested, you don't have to specify an empty slice of Tests here